core/src/main/scala/com/salesforce/op/stages/impl/feature/OPMapVectorizer.scala

```diff
@@ -124,14 +123,14 @@ class IntegralMapVectorizer[T <: OPMap[Long]](uid: String = UID[IntegralMapVecto
   def setFillWithMode(shouldFill: Boolean): this.type = set(withConstant, !shouldFill)
 
   override def fillByKey(dataset: Dataset[Seq[T#Value]]): Seq[Map[String, Double]] = {
```

Comment:
nit: it looks like fillByKey in `RealMapVectorizer` and `IntegralMapVectorizer` have a very small diff. Maybe refactor.